.gitignore

```diff
@@ -1,4 +1,5 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+/TESTS
```

Comment:
Which `/TESTS` is this ignoring? I'm assuming it's not unit tests or regular tests, right?

Reply:
Would love to know this as well! If it's some type of local folder you're using to test things, there's probably a library we could use to automate whatever it is you're doing!